src/components/books/style/addBook.css

```diff
@@ -0,0 +1,55 @@
+/* @mixin inputfield {
+  padding: 10px 14px;
+  border: 1px solid #e8e8e8;
+  border-radius: 5px;
+  outline: none;
+} */
+
```

Comment:
- [ ] Let's remove commented code. It makes code hard to read and difficult to manage.